test/Microsoft.TestPlatform.AcceptanceTests/EventLogCollectorTests.cs

```diff
@@ -87,10 +85,10 @@ private string GetRunsettingsFilePath(TempDirectory tempDirectory)
         return runsettingsPath;
     }
 
-    private void VaildateDataCollectorOutput(TempDirectory tempDir)
+    private void VaildateDataCollectorOutput(TempDirectory TempDirectory)
```

Comment:
I don't think you want this rename. Also, do we want to give the possibility to pass a temp dir or shall we use the base property directly within the method?